templates/index.html

```diff
@@ -40,8 +40,14 @@
 <div id="app">
     <div class="navbar navbar-fixed-top">
         <div class="navbar-inner" id="step1Component" style="vertical-align:middle; line-height: 40px; {{ '; display: none' if map_only }}">
-            <div style="float: left; visibility: hidden">
-                <span style="border-right: 1px solid #ddd; margin-left: 8px; padding-right: 8px; cursor: pointer">התחבר</span>
+            <div style="float: left;">
+                <span id="connection_status" style="border-right: 1px solid #ddd; margin-left: 8px; padding-right: 8px; cursor: pointer" onclick="app.LoginDialogLoad()">
+                    {% if current_user.is_authenticated %}
+                        Hi {{ current_user.nickname }}
```

Comment:
Messages should be translated, now that we have support for that. Use `gettext` and add the translation according to this: https://github.com/hasadna/anyway/blob/dev/TRANSLATE.md